generator/renderer.go

```diff
@@ -466,6 +466,7 @@ var symbolMap = []string{
 	"+", "Plus",
 	"?", "Question",
 	"$", "Dollar",
+	".", "Dot",
```

Comment:
BTW if you open a new PR with only this fix, and the comment aboive.  I will approve it right the way.